src/vcf/record.cpp

```diff
@@ -204,7 +204,9 @@ namespace ebi
                         check_field_cardinality(field.second, values, key_values["Number"], 2); // TODO Assumes ploidy=2
                         check_field_type(field.second, values, key_values["Type"]);
                     } catch (Error *ex) {
-                        throw new InfoBodyError{line, "Info " + key_values["ID"] + "=" + ex->get_raw_message()};
+                        throw new InfoBodyError{line,
+                                                "Info " + key_values["ID"] + "=" + ex->get_raw_message(),
```

Comment:
Would this construct a message like the following?

> Info AC = AC does not match the meta specification Number=A, expected 2 values

In that case, just forwarding the original exception could be enough.